flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/dataview/StateMapView.java

```diff
@@ -86,21 +89,33 @@ public EV get(EK key) throws Exception {
 
         @Override
         public void put(EK key, EV value) throws Exception {
+            if (key == null) {
```

Comment:
nit: could we refactor this check null check into a method that they all call please. 
Do we support null values in the map?

Reply:
Null values are supported. As written in the JavaDocs of MapView.